docs/design.md

```diff
@@ -84,6 +84,17 @@ Tableau capabilities are Boolean settings you can use to tune many aspects of yo
 
 For information on common capabilities and how they are used, see [Capabilities]({{ site.baseurl }}/docs/capabilities).
 
+## JDBC Driver Class Isolation
```

Comment:
Is there any user documentation or a KB article that also includes this information.  The Connector SDK should be a supporting document and not the definitive source for this information.  If so can we link to it.

Reply:
Will add the link to the user documentation